Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
Actually, could you combine the "bullet list" you added with the "Prerequisites" section above? It's a bit confusing how they are separated. I'm fine with leaving the explicit instructions for Ubuntu 14.04, but they could probably be simplified, something like:

1. Install pre-requisites listed above.
  * **\<copy-paste-helper-here\>**
2. `git clone git@github.com:dotnet/corefx`
3. `cd corefx`
4. `./build.sh`